### PR TITLE
Cy 2012 error removing rabbit node

### PR DESCRIPTION
**Pull request by glukhman**

### CODE REVIEW

File: cosmo_tester/test_suites/cluster/broker_management_test.py [hunk 1]
```diff
@@ -266,6 +267,32 @@ def test_remove(brokers, logger):
     get_cluster_listing([brokers[0], brokers[2]])
     logger.info('Dead node removed successfully.')
 
+    logger.info('Attempting to remove dead broker from manager.')
```
Comment (tyacbovi) [inline cosmo_tester/test_suites/cluster/broker_management_test.py]: add "a broker"

File: cosmo_tester/test_suites/cluster/broker_management_test.py [hunk 2]
```diff
@@ -234,8 +234,9 @@ def test_add(brokers, logger):
     logger.info('Correct error trying to join unreachable node.')
 
 
-def test_remove(brokers, logger):
+def test_remove(brokers3_and_manager, logger):
```
Comment (tyacbovi) [inline cosmo_tester/test_suites/cluster/broker_management_test.py]: add a new test

File: cosmo_tester/test_suites/cluster/broker_management_test.py (hunk 1 above)
Comment (tyacbovi) [inline cosmo_tester/test_suites/cluster/broker_management_test.py]: change the log msg

File: cosmo_tester/test_suites/cluster/broker_management_test.py [hunk 3]
```diff
@@ -266,6 +267,32 @@ def test_remove(brokers, logger):
...
+            'cfy cluster brokers remove {node} 2>&1 || true'.format(
+                node=brokers[1].hostname)
+        ).lower()
+        assert 'removed successfully' in result
+    logger.info('Dead broker removed successfully from manager.')
+
+    logger.info('Manager status test with one dead rabbit.')
+    with manager.ssh() as manager_ssh:
+        result = manager_ssh.run(
+            'cfy status 2>&1 || true'
```
Comment (tyacbovi) [inline cosmo_tester/test_suites/cluster/broker_management_test.py]: use status --json

File: cosmo_tester/test_suites/cluster/broker_management_test.py [hunk 4]
```diff
@@ -301,6 +305,50 @@ def test_remove(brokers, logger):
...
+            'cfy cluster brokers remove {node} 2>&1 || true'.format(
+                node=brokers[1].hostname)
+        ).lower()
+        assert 'removed successfully' in result
+    logger.info('Dead broker removed successfully from the manager.')
+
+    logger.info('Manager status test with one dead rabbit.')
+    with manager.ssh() as manager_ssh:
+        result = manager_ssh.run('cfy status --json 2>&1 || true')
+        assert json.loads(result.strip('\033[0m'))['status'] == 'OK'
```
Comment (glukhman) [inline cosmo_tester/test_suites/cluster/broker_management_test.py]: `strip('\033[0m') ` is for removing the logging.RESET_SEQ characters from the result

File: cosmo_tester/test_suites/cluster/broker_management_test.py [hunk 5]
```diff
@@ -301,6 +305,50 @@ def test_remove(brokers, logger):
...
+    manager.run_command(
+        'cfy cluster brokers add {name} {ip} -n "{net}"'.format(
+            name=brokers[1].hostname,
+            ip=str(brokers[1].private_ip_address),
+            net=json.dumps({'default': str(brokers[1].private_ip_address)}),
+        )
+    )
+    logger.info('Target broker added to the manager.')
+
+    logger.info('Attempting to remove a dead node.')
```
Comment (tyacbovi) [inline cosmo_tester/test_suites/cluster/broker_management_test.py]: add "a dead node from the RabbitMQ cluster"

**Pull request merged**